lib/open_project/gitolite/config/redmine_config.rb

```diff
@@ -8,6 +8,18 @@ def gitolite_use_sidekiq?
       end
 
 
+      def sidekiq_available?
+        @sidekiq_available ||=
```

Comment:
I believe we should remove this. If you want to introduce sidekiq in OpenProject, you need to replace the entire core, otherwise you will run into lots of issues. It shouldn't be part of the plugin.